### DISCUSSION TITLE
Getting Current Office Color and BaseLayerLuminance when using FluentDesignTheme

### DISCUSSION THREAD

**Discussion by AathifMahir** [microsoft/fluentui-blazor#1128] [Q&A] [answered]
How to get the Current Office Color and BaseLayerLuminance when using FluentDesignTheme, I found that GlobalState, BaseLayerLuminance and AccentColorBase Doesn't Update After Changing the Theme, These Always Stays at Default

On my case, I have BlazorApexChart, Which Get's the Current Theme and Color from BaseLayerLuminance and AccentColorBase, But After using FluentDesignTheme, it always Stays at Default Value

**Comment by dvoituron** [comment]
You should find out how to use `FluentDesignTheme` in this video. 
To retrieve the **mode** and **color**, you can add this component to your page and bind these two attributes to your local variables.

https://youtu.be/EyGKOPiT4z0

On the other hand, this component is not linked to the `GlobalState` which has its own management based on [FluentDesignTokens](https://www.fluentui-blazor.net/DesignTokens)

**Reply by AathifMahir** [reply, marked as answer]
Thanks Sorted the issue, Seems like I was Setting the Theme on OnInitialized, Instead I Set the BlazorApexChart Theme on OnAfterRender, Therefore FluentDesignTheme is Fully Rendered and Outputs the Current Theme Values
